assets/stylesheets/common/discourse-calendar.scss

```diff
@@ -281,10 +270,9 @@ $bg-in-working-hours: dark-light-choose(
   color: #069;
   opacity: 0.8;
   background: svg-uri(
-      "<svg xmlns='http://www.w3.org/2000/svg' width='2' height='2'><rect fill='#{$tertiary}' x='0' y='0' width='2' height='2'/></svg>"
+      "<svg xmlns='http://www.w3.org/2000/svg' width='2' height='2'><rect fill='#006699' x='0' y='0' width='2' height='2'/></svg>"
```

Comment:
My suggestion would be to replace this background with a transparent one, adding `position: relative` and then defining:

```css
.group-timezones-slider::before {
  background: var(--tertiary);
  content: "";
  display: block;
  height: 2px;
  position: absolute;
  top: 10px;
  width: 100%;
  z-index: -1;
}
```

Reply:
Excellent suggestion @CvX, thanks, I have added it.